js/_website/generate_jsons/src/changelog/__init__.py

```diff
@@ -0,0 +1,28 @@
+import os
+import json
+import re
+
+GRADIO_DIR = "../../../"
```

Comment:
Can we make these absolute paths relative to the files location rather than cwd. You get different result depending on where you run them from (cwd).